src/AppBundle/Controller/DepartmentController.php

```diff
@@ -12,49 +12,32 @@ class DepartmentController extends Controller
 {
     public function showAction()
     {
-        if ($this->get('security.context')->isGranted('ROLE_ADMIN')) {
-            // Finds all the departments
-            $allDepartments = $this->getDoctrine()->getRepository('AppBundle:Department')->findAll();
-
-            // Return the view with suitable variables
-            return $this->render('department_admin/index.html.twig', array(
-                'departments' => $allDepartments,
-            ));
-        } else {
-            return $this->redirect($this->generateUrl('home'));
-        }
+        return $this->render('department_admin/index.html.twig', array());
     }
 
-    public function createDepartmentAction(request $request)
+    public function createDepartmentAction(Request $request)
     {
+        // Create a new Team entity
+        $department = new Department();
 
```

Comment:
Relevant avsnitt fra Clean Code:
> Why am I so down on comments? Because they lie. Not always, and not intentionally, but too often. The older the comment is, and the farther away it is from the code it describes, the more likely it is to be just plain wrong. The reason is simple. Programmer can't realistically maintain them.